### PR TITLE
Database week2

### PR DESCRIPTION
**Pull request by masumesani**

### CODE REVIEW

File: databases/week2/homeworks.sql [hunk 1]
```diff
@@ -41,4 +40,39 @@ alter table class drop column status2;
...
+-- 
+--Get all the tasks assigned to users whose email ends in @spotify.com
+use hyf_lesson2;
+-- we could have used * but we choose to select only title and description of the task
+select  task.title,task.description,user_task.user_id, user.email
+from task 
+inner join user_task on user_task.task_id = task.id 
+inner join user on user.id = user_id
+where email regexp "@spotify.com$"  
+
```
Comment (babakbandpey) [inline databases/week2/homeworks.sql]: using regex, wowwww......., impressive

File: databases/week2/meal_application.sql [hunk 2]
```diff
@@ -0,0 +1,154 @@
...
+--Meal
+
+--Get all meals
+select * from Meal;
+--Add a new meal
+insert into Meal 
+values(default, ' Zereshk Polo','Barberry Rice','Copenhagenmm',null,null,5,null)
+-- Get a meal with any id, fx 1
+select * from Meal where Meal.id = 2;
+--Update a meal with any id, fx 1. Update any attribute fx the title or multiple attributes
```
Comment (babakbandpey) [inline databases/week2/meal_application.sql]: You have just inserted one row, and you are querying with `where Meal.id = 2`. Why is that?

File: databases/week2/meal_application.sql [hunk 3]
```diff
@@ -0,0 +1,154 @@
...
+
+-- create model
+use meal_share;
+
+create table Meal(
+	id INt auto_increment,
+	title varchar(30) NOT NULL,
+	description text, 
+	location varchar(50),
+	serving_time datetime,
```
Comment (babakbandpey) [inline databases/week2/meal_application.sql]: According to the data model: [https://dbdiagram.io/d/5d5bff66ced98361d6ddc18c](https://dbdiagram.io/d/5d5bff66ced98361d6ddc18c) there is no `serving_time `

File: databases/week2/meal_application.sql [hunk 4]
```diff
@@ -0,0 +1,154 @@
...
+select * from Meal;
+--Add a new meal
+insert into Meal 
+values(default, ' Zereshk Polo','Barberry Rice','Copenhagenmm',null,null,5,null)
+-- Get a meal with any id, fx 1
+select * from Meal where Meal.id = 2;
+--Update a meal with any id, fx 1. Update any attribute fx the title or multiple attributes
+UPDATE Meal 
+SET title = 'Fesenjan', description= 'Pomegranate Walnut Stew', location = 'Copenhagen',
+serving_time = '2020-02-02',
```
Comment (babakbandpey) [inline databases/week2/meal_application.sql]: `serving_time ` needs to be removed according to the data model diagram

File: databases/week2/meal_application.sql [hunk 5]
```diff
@@ -0,0 +1,154 @@
...
+max_reservation = 5,
+price = 125
+where id=1;
+-- 
+update Meal  
+Set title= 'Bademjan',
+description='Eggplant And Tomato Stew',
+max_reservation = 4,
+serving_time = '2020-03-10'
```
Comment (babakbandpey) [inline databases/week2/meal_application.sql]: `serving_time ` needs to be removed here

File: databases/week2/meal_application.sql [hunk 6]
```diff
@@ -0,0 +1,154 @@
...
+--Delete a meal with any id, fx 1
+delete from Meal 
+where id=3 or id =4;
+
+
+--Reservation
+--Get all reservations
+select * from Reservation;
+--Add a new reservation
+values(default,4,2,default);
```
Comment (babakbandpey) [inline databases/week2/meal_application.sql]: The insert statement is incomplete `insert into reservation` is missing

**Comment by masumesani** [comment]
> Hello Masume,
> Good job so far, but I need you to change what I have commented on.
> 
> Another request I have is to change to ids that you use int the queries accordingly, so the queries return or delete or update an existing row. If you stick to `id=1` the issue will be resolved. Thank you in advance.

Hi Babak

**Pull request merged**